go/ioutil2/ioutil_test.go

```diff
@@ -0,0 +1,17 @@
+package ioutil2
+
+import (
+	"os"
+	"testing"
+)
+
+func TestWrite(t *testing.T) {
+	fname := "/tmp/atomic-file-test.txt"
```

Comment:
I think it's better to generate a random name. What if I happen to have a file named /tmp/atomic-file-test.txt that I always keep owned by root with 0400?